client/package.json

```diff
@@ -21,7 +21,9 @@
     "react-hotkeys": "^0.9.0",
     "react-redux": "^5.0.2",
     "redux": "^3.6.0",
-    "redux-devtools-extension": "^2.0.0"
+    "redux-devtools-extension": "^2.0.0",
+    "underscore": "^1.8.3",
+    "fast-diff": "^1.1.1"
```

Comment:
@sdee fast-diff put in package.json for client but used by server. It needs to go in the other package.json